pkgs/development/haskell-modules/configuration-tensorflow.nix

```diff
@@ -62,6 +62,30 @@ in
     { mkDerivation, base, bytestring, Cabal, cryptonite, directory
     , filepath, HTTP, network-uri, stdenv
     }:
+
+    let
+      urlPrefix = "http://yann.lecun.com/exdb/mnist/";
+
+      # File names relative to 'urlPrefix' and their sha256.
+      fileInfos = [
```

Comment:
I think using an attribute set here which maps file names to their hashes is a bit clearer and safer to use.